src/app/services/auth.service.ts

```diff
@@ -1,16 +1,18 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject, Observable } from 'rxjs';
-import { User } from '../models';
+import {User} from "../models/user";
 import {CommonHttpService} from "./common-http.service";
 
 @Injectable({ providedIn: 'root' })
-export class AuthenticationService {
+export class AuthService {
 
   private userSource: BehaviorSubject<User>;
   public user$: Observable<User>;
-
+  public user: User = new User(2);
```

Comment:
regular `user` property is not needed. `userSourve` should be your user, you can even name it `user`